easybuild/easyconfigs/p/Perl-bundle-CPAN/Perl-bundle-CPAN-5.36.1-GCCcore-12.3.0.eb

```diff
@@ -1768,6 +1768,7 @@ exts_list = [
         'checksums': ['9dfd6ca2822724347e0eb6759d00709425814703ad5c66bdb6214579868bcac4'],
     }),
     ('Time::HiRes', '1.9764', {
+        'runtest': False,  # atime tests are useless and sometimes break
```

Comment:
I think it's good to refer to the issue https://github.com/easybuilders/easybuild-easyconfigs/issues/20145 in the comment. That provides more context for why this was disabled.